pinot-common/src/main/java/com/linkedin/pinot/common/response/broker/GroupByResult.java

```diff
@@ -50,4 +50,9 @@ public void setValue(Serializable value) {
   public void setGroup(List<String> group) {
     _group = group;
   }
+
+  @Override
+  public String toString() {
+    return _group + ": " + _value.toString();
```

Comment:
The toString should not be used in any part of the real production code. It's only used for related test codes. The calling should have a sense of how much the output is and whether to use it or not. We can surely attach a warning in the comment of this function.